runtime/sema/type.go

```diff
@@ -4366,6 +4366,11 @@ func (t *CompositeType) IsStorable(results map[*Member]bool) bool {
 		return false
 	}
 
+	// Native/built-in types are not storable for now
+	if t.Location == nil {
```

Comment:
since making it non-storable makes it non-importable too, should we also make it non-exportable (non externally returnable), just to be consistent?

Reply:
That's a good point! I'll try to make the change and see 👍

Reply:
Oh, I see. Yeah, that's a fair point! Maybe it is better to leave it this way then.
I guess we will be anyway allowing them to be passed as arguments, and this limitation is only temporary.

Thanks for checking on that!

Reply:
Right, the non-importability should ideally only be temporary. I've opened #717 to make the types importable again